### PR TITLE
shorten abstract

### PR DESCRIPTION
**Pull request by Logende**
I've shortened the abstract

### CODE REVIEW

File: paper/implementation.tex [hunk 1]
```diff
@@ -75,11 +75,11 @@ \subsubsection{Toolbar functionalities}
     For the File Editor, this opens a dialog to open a file from the local file system.
     For the Schema Editor, this comes with four sub-options:
     \begin{itemize}
-        \item JSON Schema Store: Choose a schema from the JSON Schema Store, which provides a list of available schemas.
+        \item JSON Schema Store: Choose a schema from the JSON Schema Store, the biggest online collection of JSON schemas.
 
-        \item From Our Examples: Access four example schemas provided within the tool.
+        \item From Our Examples: Access one of the example schemas inbuilt into the tool.
```
Comment (PaulBredl) [inline paper/implementation.tex:80]: The option don't align with the text in the gui
image

**Review by PaulBredl — APPROVED** [review]
Good work with the abstract!

File: paper/implementation.tex (hunk 1 above)
Comment (Logende) [reply paper/implementation.tex]: @PaulBredl better now?

Comment (PaulBredl) [reply paper/implementation.tex]: I actually ment the text isn't the same, and I thought the item elements would represent the actual menu text. But I guess the meaning is clear

**Review by PaulBredl — APPROVED** [review]

**Pull request merged**